fahari/mle/models.py

```diff
@@ -0,0 +1,85 @@
+from django.contrib.gis.db import models
+from django.urls.base import reverse
+from django.utils import timezone
+
+from fahari.common.models import AbstractBase, Facility
+
+
+class FacilityNetworkCondition(AbstractBase):
```

Comment:
Could you change this name to something more descriptive? What is a network condition?

Reply:
Suggest a generic name that will tell whether a facility has internet and/or networked